src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -94,24 +94,22 @@ def _get_histogram_min(histogram: Histogram):
         # max, respectively.
         return histogram_sum
 
+    # iterate all buckets to find the first bucket with count > 0
     for index in range(0, len(histogram.bucket_counts)):
+        # the current bucket contains something.
         if histogram.bucket_counts[index] > 0:
             if index == 0:
-                # If we are in the first bucket, use the upper bound (which
-                # is the lowest specified bound overall) otherwise this would
-                # be -Inf, which is not allowed. This is not quite correct,
-                # but the best approximation we can get at this point. This
-                # might however lead to a min that is bigger than the sum,
-                # therefore we return the min of the sum and the lowest
-                # bound. Choose the minimum of the following three: - The
-                # lowest boundary - The sum (smallest if there are multiple
-                # negative measurements smaller than the lowest boundary) -
-                # The average in the bucket (smallest if there are multiple
-                # positive measurements smaller than the lowest boundary)
-                return min(
-                    min(histogram.explicit_bounds[index], histogram_sum),
-                    histogram_sum / histogram_count
-                )
+                # In the first bucket, (-Inf, firstBound], use firstBound (
+                # this is the lowest specified bound overall). This is not
```

Comment:
```suggestion
                # In the first bucket, (-Inf, firstBound], use firstBound 
                # (this is the lowest specified bound overall). This is not
```

Reply:
Thx! Applied in [7920eba](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/24/commits/7920eba0f4e703df5ca9e36a147758809cb8d26c) :slightly_smiling_face: